docs/Installing.asciidoc

```diff
@@ -1036,15 +1036,15 @@ updates and if it does it will upgrade the whole system. This approach is
 independent of `openQA-auto-update` but can be used complementary. The
 configuration is analogous to `openQA-auto-update`.
 
-==== Migrating from older databases
+=== Migrating from older databases
```

Comment:
Level 3 is still not correct as we are still below "Automatic system upgrades and reboots of openQA hosts". Just make it level 2

```suggestion
== Migrating from older databases
```

same further below